lib/theme_check/checks/img_lazy_loading.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module ThemeCheck
+  class ImgLazyLoading < HtmlCheck
+    severity :suggestion
+    categories :html, :performance
+    doc docs_url(__FILE__)
+
+    def on_img(node)
+      if node.attributes["loading"]&.value != "lazy"
+        add_offense("Add loading=\"lazy\" to defer loading of images", node: node)
```

Comment:
should also check for 'auto' which was pitched originally but was determined to be a bad idea. we should flag this as well with more nuanced response